pkgs/development/python-modules/systembridgeconnector/default.nix

```diff
@@ -25,15 +25,18 @@ buildPythonPackage rec {
     owner = "timmo001";
     repo = "system-bridge-connector";
     rev = "refs/tags/${version}";
-    hash = "sha256-uqE/KJnuNii2b3geB9jp8IxaeceuZVXdol7s3hP6z/Q=";
+    hash = "sha256-AzAN7reBAI4atEFutgFrdQHFy/Qc90PQxwSaHaftn5Q=";
   };
 
   postPatch = ''
     substituteInPlace systembridgeconnector/_version.py \
       --replace-fail ", dev=0" ""
   '';
 
-  build-system = [ setuptools ];
+  build-system = [
+    incremental
```

Comment:
I think you'll have to patch out the pinning of incremental in `requirements_setup.txt`.